corehq/apps/domain/views/settings.py

```diff
@@ -584,6 +587,9 @@ def post(self, request, *args, **kwargs):
         user = User.objects.get(pk=uid)
         couch_user = CouchUser.from_django_user(user)
         clear_login_attempts(couch_user)
+        log_user_change(by_domain=couch_user.domain, for_domain=couch_user.domain, couch_user=couch_user,
```

Comment:
Won't this error if `couch_user` is a `WebUser`?

Reply:
yep you're right. Made it so it only records domain for mobile workers